src/gui/mrview/window.cpp

```diff
@@ -2012,7 +2013,7 @@ namespace MR
             return;
           }
 
-          assert ("shouldn't reach here!" && false);
+          assert (opt.opt->is ("info") or opt.opt->is ("debug") or ("shouldn't reach here!" && false));
```

Comment:
Struggling a little to understand how this even compiles...? What are these 'or' statements doing in there? This is supposed to be C++, not python... 😕